webapp/src/app/mentor/messages/messages.component.html

```diff
@@ -1,27 +1,60 @@
 <div class="flex flex-col gap-4">
-  @for (message of messages(); track message.id) {
-    <div class="flex w-full" [ngClass]="{ 'justify-end': message.sender === Message.SenderEnum.User, 'justify-start': message.sender === Message.SenderEnum.Mentor }">
-      <div class="flex space-x-2 md:w-3/5" [ngClass]="{ 'flex-row-reverse': message.sender === Message.SenderEnum.User }">
-        @if (message.sender === Message.SenderEnum.Mentor) {
-          <div class="mr-2 flex flex-col">
-            <div class="w-10 h-10 bg-transparent border-2 border-cyan-500 rounded-full flex items-center justify-center">
-              <lucide-angular [img]="BotMessageSquare" class="size-6 text-cyan-500" />
+  @if (isLoading()) {
+    <div class="flex w-full">
```

Comment:
A loop should be able to simplify this and avoid copy-pasted code.

You can work with `i % 2 === 0` for left/right-alignment.